aten/src/ATen/core/ivalue_inl.h

```diff
@@ -1151,7 +1139,7 @@ IValue from_(T x, std::false_type) {
 
 template <typename T>
 IValue from(T x) {
-  return detail::from_(std::move(x), detail::has_constructor<T>{});
+  return detail::from_(std::move(x), typename std::is_constructible<IValue, T>::type{});
```

Comment:
Is it newly possible to use `is_constructible` instead of `has_constructor`, or is this just cleanup?

Reply:
the old implementation ran into compiler bugs on nvcc but is_constructible would always have worked, so it's just a cleanup.